components/butterfly/src/rumor/mod.rs

```diff
@@ -28,6 +28,11 @@ pub mod service;
 pub mod service_config;
 pub mod service_file;
 
+pub use self::election::Election;
```

Comment:
I find this kind of re-exporting confusing, because my imports no longer map to where I can explore the source on disk. I'm fine with it, and it's a common enough pattern, but I think the verbose mechanism is better.